### ISSUE TITLE
Any chance of a kern for "fa" and "Fa"

### ISSUE THREAD

**Issue by revk** [ipython/xkcd-font#41]
The spacing on `fa` and `Fa` are messy, any change of a kern setting to tighten slightly?

I have no idea how to do that else I would do a pull request.

**Comment by pelson** [comment]
Agreed. In general it is the right-hand spacing around `F` `f` and left-hand spacing around `g` which sticks out the most. 
[demo](https://ipython.org/xkcd-font/preview.html?text=fa%20fb%20fc%20fd%20fe%20ff%20fg%20fh%20fi%20fj%20fk%20fl%20fm%20fn%20fo%20fp%20fq%20fr%20fs%20ft%20fu%20fv%20fw%20fy%20fz%0A%0AFa%20Fb%20Fc%20Fd%20Fe%20Ff%20Fg%20Fh%20Fi%20Fj%20Fk%20Fl%20Fm%20Fn%20Fo%20Fp%20Fq%20Fr%20Fs%20Ft%20Fu%20Fv%20Fw%20Fy%20Fz%0A%0Agag%20gbg%20gcg%20gdg%20geg%20gfg%20ggg%20ghg%20gig%20gjg%20gkg%20glg%20gmg%20gng%20gog%20gpg%20gqg%20grg%20gsg%20gtg%20gug%20gvg%20gwg%20gyg%20gzg).

**Comment by revk** [comment]
Thanks.
